Лобанов К.И/coinGame.js

```diff
@@ -0,0 +1,49 @@
+const readline = require('readline');
+const fs = require('fs');
+
+function random(min, max) {
+  var rand = min - 0.5 + Math.random() * (max - min + 1)
```

Comment:
Не стоит смешивать var и let/const
```suggestion
  let rand = min - 0.5 + Math.random() * (max - min + 1)
```